### PR TITLE
[4.0] Add 'watch' task to Grunt

### PR DESCRIPTION
**Pull request by C-Lodder**
Pull Request for Issue #15971

### Summary of Changes

Add a watch task so that compiling occurs automatically when making changes to SCSS files

@ciar4n @dgt41 Note to both. Once this is merged and you've fetched, please run `npm i`

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -15,7 +15,7 @@
     "autoprefixer": "^6.7.7",
     "grunt": "latest",
     "grunt-cli": "latest",
-    "grunt-contrib-clean": "latest",
+    "grunt-contrib-watch": "^1.0.0",
```
Comment (dgt41) [inline package.json]: we still need the clean

Comment (C-Lodder) [inline package.json]: oops, fixed.

**Comment by C-Lodder** [comment]
@wilsonge can be merged

**Comment by wilsonge** [comment]
Please add documentation for this in the grunt readme

**Pull request merged**